src/components/GameItem.js

```diff
@@ -3,24 +3,62 @@ import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
+// ### Wave 2 - Spotting Items
+// For this wave you will begin to implement the game logic by allowing the player to spot items as they scroll by. Players can "spot" an item by clicking on it, and the game gives immediate feedback on whether the item was litter.
+//
+// * When a player clicks on a game item that has not yet been clicked upon, the item displays either:
+//     * a green check, if the item has type 'litter'
+//     * a red X, otherwise
+// * When a player clicks on a game item that has already been clicked upon, nothing changes with the item's display
+//
+// Each `GameItem` component should track whether it has been spotted using an event handler. You will need to update the `render` function as well to use the appropriate CSS classes to indicate whether the spotted item was litter or not.
+
 class GameItem extends Component {
-  propTypes = {
+
+  constructor(props) {
+    super(props);
+    this.state = props.state; //change prop state
```

Comment:
I would very much never call a prop `state`, because it could be confusing.  Also you never passed a prop into `GameItem` from `App`, so `props.state` is undefined.